src/Parser/RestoDruid/Modules/Traits/SeedsOfTheWorldTree.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import calculateEffectiveHealing from 'Parser/Core/calculateEffectiveHealing';
+import SPELLS from 'common/SPELLS';
+
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage } from 'common/format';
+
+const SEEDS_OF_THE_WORLD_TREE_HEALING_INCREASE = 0.08;
```

Comment:
Seeds of the World Tree doesn't increase Living Seed by 8%, it increases _the portion of the initial heal that goes into the Living Seed by 8%_. Living Seed baseline is 25%, this constant should actually be (0.08 / 0.25). Inspection of Living Seed healing amounts confirms this.

Reply:
Done